lotusrb.gemspec

```diff
@@ -22,6 +22,7 @@ Gem::Specification.new do |spec|
   spec.add_dependency 'lotus-utils',      '~> 0.3', '>= 0.3.4'
   spec.add_dependency 'lotus-router',     '~> 0.2', '>= 0.2.1'
   spec.add_dependency 'lotus-controller', '~> 0.3', '>= 0.3.2'
+  spec.add_dependency 'lotus-model',      '~> 0.2.2', '>= 0.2.2'
```

Comment:
Sure, I should've not committed this file because I don't know what put on it.
I'm using local reference on my Gemfile but I didn't commit because it's referring local repo. I'll change both to point to your branch.